lib/app/modules/cart/cart_store.dart

```diff
@@ -1,16 +1,40 @@
+import 'package:flutter_modular/flutter_modular.dart';
 import 'package:mobx/mobx.dart';
 import 'package:pscomidas/app/global/models/entities/item.dart';
-
+import 'package:pscomidas/app/global/models/entities/order.dart';
+import 'package:pscomidas/app/global/repositories/order/order_repository.dart';
 part 'cart_store.g.dart';
 
 class CartStore = _CartStoreBase with _$CartStore;
 
 abstract class _CartStoreBase with Store {
+  final OrderRepository orderRepository = Modular.get<OrderRepository>();
   @observable
   List<Item> itens = [];
 
   @action
   void addItem(Item item) {
     itens.add(item);
   }
+
+  @computed
+  double get total {
+    double subtotal = 0;
+    itens.forEach((item) {
```

Comment:
o uso de forEach() faz parecer que o código é mais complicado do que ele é de verdade, por isso a IDE recomenda não usar, além disso ele dificulta a detecção de erros na lógica e pode prejudicar o desenvolvimento. você pode usar um loop 'for'